changelog/formatting_floats_in_CTFE.dd

```diff
@@ -1,4 +1,4 @@
-`float` and `double` values can be formatted at compile time
+`float`, `double` and certain `real` values can be formatted at compile time
```

Comment:
64-bit and 80-bit `real`s accounts for 100% of the platforms supported by dmd, so I think the note below is sufficient.

```suggestion
`float`, `double` and `real` values can be formatted at compile time
```

Reply:
I suggest "Floating point numbers can be ..."? No need to list all of them...